subtrack.MAUI/Pages/Index.razor

```diff
@@ -15,7 +15,7 @@
             <h6 class="pt-1">Monthly</h6>
         </div>
         <div class="col">
-            <h5>@($"{_subscriptionsCost:C}")</h5>
+            <h5>@($"~{Math.Round(_subscriptionsCost, 2):C}")</h5>
```

Comment:
Cant we round the subscription cost where we fetch it?

Reply:
Perhaps. But it's not a calculation concern, it's a display concern and should remain here in the view.

Reply:
> Perhaps. But it's not a calculation concern, it's a display concern and should remain here in the view.

This class is the view and as long as the value isnt used elsewhere (i dont have any plans for that) theres no reason to add code next to markup